README.md

```diff
@@ -161,6 +196,11 @@ We use the kubernetes style for this. For example, kubernetes -> k(# of characte
 
 For some specific resources, we have a 32 characters limit. If the identifier length exceeds this number after compression, we remove the middle part and replace it for a hash made up from the string itself. 
 
+## Note about tagging
+
+There's the option to add any kind of defined tag's to each grouping module. Will be added to the commons tagging.
+An example of how to set them: `{"key1": "value1", "key2": "value2"}`'
+
 ### S3 buckets naming
 
 Buckets names can be made of up to 63 characters. If the length allows us to add -tf-state, we will do so. If not, a simple -tf will be added.
```

Comment:
1. Buckets names can be ~~made of~~ up to 63 characters long.
2. If length allows, `-tf-state` will be suffixed to the name. Otherwise, only `-tf` will be added.
3. In all cases, the name hashing described above will be used to keep the lengths within limit.